CONTRIBUTORS.MD

```diff
@@ -0,0 +1,4 @@
+#CONTRIBUTORS
```

Comment:
I'm not certain we need this file because you can already see contributors here https://github.com/Recycle-Buddy/front-end/graphs/contributors

Reply:
I mainly wanted to add it as a really easy way for people to make their first contribution. A way for someone to just practice fork/clone/code/commit/PR/etc. I could have definitely made this more clear though. Thanks for letting me know about this page though, I have yet to check out all of the buttons and pages of github :)

Reply:
cool, it is good to try to get everyone to get a commit in. I think it will be easier once we have some foundations in; it's always easier to lay the second brick.